css/modern.css

```diff
@@ -1528,6 +1620,16 @@ div.cellphone-container {
 }
 
 
+.settings-table tbody td.selected, .settings-table tbody td.selected:hover {
+    background-color: var(--active) !important;
+}
+.settings-table tbody td {
+    background-color: var(--primary) !important;
+}
+.settings-table tbody td.realm-disabled, .settings-table tbody td.realm-disabled:hover, .settings-table tbody td.area-disabled {
+    background-color: transparent !important;
```

Comment:
Transparent -> none?

Reply:
No idea if transparent is any different to none.